File: src/test/java/s2/TestS2.java
package s2;

import oopsla25.TestUtil;
import org.junit.Test;
import org.scribble.ext.gt.cli.GTCommandLine2;

public class TestS2 {

    @Test
    public void testTwoPartyChoice() {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\TwoPartyChoice.scr");
    }

    @Test
    public void testTwoPartyChoiceNonDirected() {
        TestUtil.testBad(TestUtil.BASE_PATH + "\\s2\\bad\\TwoPartyChoiceNonDirected.scr");
    }

    @Test
    public void testTwoPartyMC() {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\TwoPartyMC.scr");
    }

    @Test
    public void testTimeout() {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\Timeout.scr");
    }
}

File: src/test/scala/com/github/rhu1/gt/test/oopsla25/s2/TestS2.scala
package com.github.rhu1.gt.test.oopsla25.s2;

import com.github.rhu1.gt.test.oopsla25.TestUtil
import org.scalatest.funsuite.AnyFunSuite;


class TestS2 extends AnyFunSuite {

    test("TwoPartyChoice") {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\TwoPartyChoice.scr");
    }

    test("TwoPartyChoiceNonDirected") {
        TestUtil.testBad(TestUtil.BASE_PATH + "\\s2\\bad\\TwoPartyChoiceNonDirected.scr");
    }

    test("TwoPartyMC") {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\TwoPartyMC.scr");
    }

    test("Timeout") {
        TestUtil.testGood(TestUtil.BASE_PATH + "\\s2\\good\\Timeout.scr");
    }
}
